homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -68,6 +68,8 @@ async def async_lock(self, **kwargs: Any) -> None:
             self.async_write_ha_state()
         await self.vehicle.remote_services.trigger_remote_door_lock()
 
+        self.coordinator.refresh_state_after_remote_service()
```

Comment:
Why not directly call `async_update_listeners()`?

Reply:
I wanted to make clear that the API update was already done. I've changed to `async_udpdate_listeners()` if that is more clear/consistent with other integrations.